app/src/main/res/values/strings.xml

```diff
@@ -55,7 +55,7 @@
     <string name="hint_password">Password</string>
     <string name="hint_key_password">Key password</string>
 
-    <string name="msg_about_app">A copylefted libre VNC app.\nUse any device with a VNC server on it remotely.</string>
+    <string name="msg_about_app">Use other devices and PCs that have VNC servers with this copylefted libre VNC app.\nUse, inspect, change and share — with all.</string>
```

Comment:
Please don't do this. This will lead us nowhere.

Reply:
It was self-referencing, misspelling "Open Source" and then stating it operates on Android for no reason.
Now, it is the same as what works in Remmina.

Reply:
I was referring to how you keep changing the original string, simply ignoring my suggestions.